pkg/batchd/cache/cache.go

```diff
@@ -673,7 +674,7 @@ func (sc *SchedulerCache) Snapshot() *CacheSnapshot {
 	return snapshot
 }
 
-func (sc SchedulerCache) String() string {
+func (sc *SchedulerCache) String() string {
```

Comment:
That should not be a pointer :)

Reply:
Thanks for the review , @k82cn ! Hmm, if this is not a pointer receiver, why do we call `sc.Mutex.Lock` inside `String`? 

https://medium.com/golangspec/detect-locks-passed-by-value-in-go-efb4ac9a3f2b

Reply:
Thanks for the review , @k82cn ! Hmm, if this is not a pointer receiver, why do we call sc.Mutex.Lock inside String?

https://medium.com/golangspec/detect-locks-passed-by-value-in-go-efb4ac9a3f2b

Reply:
Good point ! LGTM :)